app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide :title, "Home" %>
+
+<h1>Sample App</h1>
+<p>
+  This is the home page for the
+  <%= link_to "Ruby on Rails Tutorial", Settings.links.rails_tutorial.home_page %>.
+</p>
```

Comment:
add new line at end of file, please check with another files

Reply:
done sir